Makefile

```diff
@@ -140,17 +141,25 @@ ui:
 .PHONY: install
 install: check-dcos
 	$(DCOS_CMD) package install --yes marathon-lb;\
-	$(DCOS_CMD) package install --yes kubernetes --package-version=$(KUBERNETES_FRAMEWORK_VERSION) --options="$(PATH_TO_PACKAGE_OPTIONS)";\
-	$(DCOS_CMD) marathon app add "$(PWD)/.deploy/kubeapi-proxy.json"
+	$(DCOS_CMD) marathon app add "$(PWD)/.deploy/kubeapi-proxy.json" || true
+	$(DCOS_CMD) package install --yes kubernetes --package-version="$(KUBERNETES_FRAMEWORK_VERSION)";\
+	sleep 30
+	$(DCOS_CMD) kubernetes cluster create --yes --options="$(PATH_TO_PACKAGE_OPTIONS)";\
+
 
 .PHONY: watch
 watch:
```

Comment:
Please rename to `watch-kubernetes-cluster`.

Reply:
would this not break previous expectations?